src/coreComponents/virtualElement/VirtualElementBase.hpp

```diff
@@ -37,21 +40,85 @@ class VirtualElementBase
   /// Default destructor.
   virtual ~VirtualElementBase() = default;
 
-  /// @brief Compute VEM projectors on the geometry.
-  virtual void computeProjectors( MeshLevel const &, localIndex const &,
-                                  localIndex const &, localIndex const & ) = 0;
+  /**
+   * @brief Get the shape function projected derivatives at a given quadrature point.
+   * @details The output contains the integral mean of derivatives.
+   * @tparam LEAF Type of the derived virtual element implementation.
+   * @param q The quadrature point index.
+   * @param gradN Return array of the shape function projected derivatives. Size will be @ref
+   * getNumSupportPoints() x 3.
+   */
+  template< typename LEAF >
```

Comment:
I don't see the utility of these functions templated on `LEAF`. They are simply calling the `LEAF` function without modification, and they cannot be called without knowing the type of `LEAF`, so why not just call the `LEAF` function directly?

Reply:
I did these methods so that the `VirtualElementBase` class resembles the `FiniteElementBase` class (see e.g. [here](https://github.com/GEOSX/GEOSX/blob/8f661c7ea0916c8c04c6068bd999fbe8042564b3/src/coreComponents/finiteElement/elementFormulations/FiniteElementBase.hpp#L466)).